stable/airflow/README.md

```diff
@@ -274,7 +274,8 @@ The following table lists the configurable parameters of the Airflow chart and t
 | `airflow.webReplicas`                    | how many replicas for web server                        | `1`                       |
 | `airflow.config`                         | custom airflow configuration env variables              | `{}`                      |
 | `airflow.podDisruptionBudget`            | control pod disruption budget                           | `{'maxUnavailable': 1}`   |
-| `airflow.secretsMapping`		           | override any environment variable with a secret	     | 				             |
+| `airflow.secretsMapping`                 | override any environment variable with a secret         |                           |
+| `airflow.pod.annotations`                | annotations for the web and scheduler pods              | `{}`                      |
```

Comment:
By standard we're using podAnnotations config

Reply:
wouldn't it make sense to have 2 types of annotation (one for workers, one for scheduler)?

Reply:
I agree: it makes sense to have different annotations for the workers - especially if adding IAM Role annotations (for kiam/kube2iam) is a common use-case.  I wouldn't want the web Pods to have the same IAM permissions as the Workers.

There is a separate set of annotations for the workers already - at line https://github.com/helm/charts/pull/11520/files#diff-57999b743b4027d904b342d1f6fbe507R283.
(Apologies if it doesn't follow the conventions - I added this in an earlier change)

How about we change to adopt the convention by using `.Values.podAnnotations` for the Web and Scheduler Pods and keep the `workers.pod.annotations` just for the workers?

Reply:
(We could also rename `workers.pod.annotations` to match the standard, but that would be a compatibility break).

Reply:
I do not have a strong opinion on it.
- it look like good practice says "podAnnotation" and that's ok for me
- we already have worker.pod.annotation so we would need to keep consistency if we were to add annotation for scheduler and web".

So I propose:
- airflow.podAnnotation: annotation for ALL pods (workers, scheduler, web)
- "worker.pod.annotation" to keep
- add "scheduler.pod.annotation" for the scheduler and "web.pod.annotation" for the web server

Reply:
Thanks for your reviews.  I have implemented the first 2 of @gsemet's proposed changes - please take another look!

Regarding the 3rd bullet above (specific annotations for the scheduler and web Pods): do we see a use-case for this?  I'd expect it to be more common to have a specific annotaiton on the workers.  If you would like I can add this, otherwise we can wait until the requirement comes up.